mesheryctl/pkg/utils/relationship.go

```diff
@@ -0,0 +1,224 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/layer5io/meshkit/encoding"
+	"github.com/layer5io/meshkit/utils"
+	"github.com/layer5io/meshkit/utils/csv"
+	"github.com/meshery/schemas/models/v1alpha3"
+	_rel "github.com/meshery/schemas/models/v1alpha3/relationship"
+	"github.com/meshery/schemas/models/v1beta1/model"
+)
+
+type RelationshipCSVHelper struct {
+	SpreadsheetID  int64
+	SpreadsheetURL string
+	Title          string
+	CSVPath        string
+	Relationships  []RelationshipCSV
+}
+type RelationshipCSV struct {
+	Model       string `json:"Model" csv:"Model"`
+	Version     string `json:"Version" csv:"Version"`
+	KIND        string `json:"kind" csv:"kind"`
+	Type        string `json:"type" csv:"type"`
+	SubType     string `json:"subType" csv:"subType"`
+	Description string `json:"metadata.description" csv:"metadata.description	"`
```

Comment:
Did you verified this tag? Does it work correctly? 
We might need an custom implementation for Unmarshal. (for xml it works, I believe but not for JSON).

Reply:
It did I'll recheck and confirm.